DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -833,8 +845,14 @@ if($resourceCreation -eq 'y') {
     Write-Host -ForegroundColor Green "Deploying resources (1/16 steps): All resources except HDInsight and Service Fabric clusters will be deployed"
     Write-Host -ForegroundColor Green "Estimated time to complete: 40 mins"
     
-    $tokens = Get-Tokens
+	$tokens = Get-Tokens
     Deploy-Resources -templateName "Resource-Template.json" -paramName "Resource-parameter.json"  -templatePath $templatePath -tokens $tokens
+	
+	if($setupAdditionalResourcesForSample -eq 'y')
+	{
+		Write-Host -ForegroundColor Green "Deploying SQL type CosmosDB for output sample"
```

Comment:
is it a good idea to show the estimated time?

Reply:
moved to deploySample.ps1. Deployment completes within already shown expected time